test/stage2/compile_errors.zig

```diff
@@ -1,8 +1,27 @@
 const TestContext = @import("../../src-self-hosted/test.zig").TestContext;
+const std = @import("std");
+
+const ErrorMsg = @import("../../src-self-hosted/Module.zig").ErrorMsg;
+
+const linux_x64 = std.zig.CrossTarget{
+    .cpu_arch = .x86_64,
+    .os_tag = .linux,
+};
 
 pub fn addCases(ctx: *TestContext) !void {
     // TODO: re-enable these tests.
     // https://github.com/ziglang/zig/issues/1364
+    ctx.addZIRError("test", linux_x64,
+        \\@noreturn = primitive(noreturn)
+        \\
+        \\@start_fnty = fntype([], @noreturn, cc=Naked)
+        \\@start = fn(@start_fnty, {
+        \\  %0 = call(%test, [])
+        \\})
+    , &[_]ErrorMsg{.{
+        .byte_offset = 118,
+        .msg = "unrecognized identifier: %test",
+    }}, &[_]ErrorMsg{}, &[_]ErrorMsg{});
```

Comment:
Thanks for implementing the test harness for these kind of tests. Would you be willing to make it even easier to make test cases?

I think this can be much simpler, a single list of error strings like this:

```zig
&[_][]const u8{
  "10:13: error: unrecognized identifier: %test",
  "15:5: error: blah",
}
```

This will map directly to the output you would get from running e.g. `zig build-obj` and so would be easier for contributors to add test cases. We have a lot to gain from making it as simple as possible to add new test cases!

Reply:
Working on this right now; figured out a way to get it done without needing a ChildProcess :)

Reply:
Fixed!